README.md

```diff
@@ -1,2 +1,135 @@
-# aaq-core
-Ask-a-question service core service
+<pre align="center" style="text-align:center; font-size: 1vw; background:none;">
+    __            __        ______
+    /""\          /""\      /    " \
+    /    \        /    \    // ____  \
+    /' /\  \      /' /\  \  /  /    )  )
+  //  __'  \    //  __'  \(: (____/ //
+ /   /  \\  \  /   /  \\  \\         \
+ (___/    \___)(___/    \___)\"____/\__\
+ASK-A-QUESTION
+</pre>
+
+<p align="center" style="text-align:center">
+<a href="https://idinsight.github.io/aaq-core/">Docs</a> |
+<a href="#features">Features</a> |
+<a href="#usage">Usage</a> |
+<a href="#architecture">Architecture</a> |
+<a href="#partners">Partners</a>
+</p>
+
+**[Ask-a-question](https://idinsight.github.io/aaq-core/) allows your beneficiaries or staff to ask question in natural language in their preferred language and receive answers from your content repository**
+
+## :woman_cartwheeling: Features
+
+#### **:question: LLM-powered search**
+
+Match your questions to content in the database using embeddings from LLMs.
+
+#### **:robot: LLM responses**
+
+Craft a custom reponse to the question using LLM chat and the content in your database
+
+#### **:speech_balloon: Deploy on whatsapp**
+
+Easily deploy using WhatsApp Business API
+
+#### **:books: Manage content**
+
+Use the Admin App to add, edit, and delete content in the database
+
+## :construction: Upcoming
+
+#### **:earth_africa: Support for low resourced language**
+
+Ask questions in local languages. Languages currently on the roadmap
+
+- [ ] Xhosa
+- [ ] Zulu
+- [ ] Hindi
+- [ ] Igbo
+
+#### **:speech_balloon: conversation capability**
+
+Refine or clarify your question through conversation
+
+#### :video_camera: multimedia content
+
+Respond with not just text but images and videos as well.
```

Comment:
This isn't on the roadmap; should this be about voicenotes?
Re: images, content could be having URLs for images which we can render on WhatsApp easily